Automattic/vip-workflow-plugin
# Move/notifications to settings

## Description

- Lint the eslint config as we skipped that last time
- Delete the usergroup module
- Delete the usergroup related code in notifications, as well as make notifications targeted at all the posts instead
- Flip the notify admin flag to true by default, so it's a good fallback

What I haven't done:

- Converted the notifications page to be react based. I'm waiting for @alecgeatches's PR to go in before doing that.

## Steps to Test

- Test each module
- Setup webhook urls
- Ensure notifications are correctly sent.

You can configure mail pit via `vip dev-env` and have emails be sent through mail pit. You'll see the url for it when you start your `dev-env`.
